backend/corpora/lambdas/api/v1/curation/collections/common.py

```diff
@@ -50,14 +50,9 @@ def reshape_for_curation_api_and_is_allowed(
     """
 
     owner = collection.pop("owner")  # Don't actually want to return 'owner' in response
-    if is_user_owner_or_allowed(token_info, owner):
-        collection["access_type"] = "WRITE"
-    elif not id_provided and collection["visibility"] == CollectionVisibility.PRIVATE:
+    if not id_provided and collection["visibility"] == CollectionVisibility.PRIVATE:
```

Comment:
this breaks [Super Curator #3](https://github.com/chanzuckerberg/single-cell-data-portal/pull/3089#issuecomment-1213348180)

Reply:
You should do as you see fit, but I think that the right way to resolve the issue here with the refactoring you're introducing in #2793 is to basically do away with the "is_allowed" portion of this method (this method was always trying to do two things which was a bad design choice on my part), move the auth logic outside of the reshaping logic (i.e., auth logic occurs in body of `...collections.actions.get` handler method, with just a single `is_super_curator(token_info)` call), and then this collection reshaping method gets called only if the user is authorized for the collection 🏄‍♂️ 🤙